bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/internal/ThingManagerImpl.java

```diff
@@ -472,6 +447,35 @@ protected synchronized void deactivate(ComponentContext componentContext) {
         readyService.unregisterTracker(this);
     }
 
+    private void thingUpdated(final Thing thing) {
+        thingUpdatedLock.add(thing.getUID());
+        AccessController.doPrivileged((PrivilegedAction<@Nullable Void>) () -> {
```

Comment:
I think that the `AccessController` can be removed, it is deprecated for removal in Java 17 anyway and I know of no code that would depend on it in openHAB. But we should probably clean-up all code that uses the `SecurityManager` after we switched to Java 17 at the end of this year.